types/types.proto

```diff
@@ -61,7 +61,8 @@ message Provider {
   option (gogoproto.compare) = true;
   bytes  address = 1 [(gogoproto.customtype)="github.com/ovrclk/akash/types/base.Bytes",(gogoproto.nullable) = false];
   bytes  owner = 2 [(gogoproto.customtype)="github.com/ovrclk/akash/types/base.Bytes",(gogoproto.nullable) = false];
-  repeated ProviderAttribute attributes = 3 [(gogoproto.nullable) = false];
+  string netaddr = 3;
```

Comment:
let's call this `host_uri` (`HostURI`), `akash_uri`, or similar.  `netaddr` kinda implies `ip:port` pair.